modApiServer/src/org/aion/api/server/rpc3/types/RPCTypesConverter.java

```diff
@@ -0,0 +1,378 @@
+            } catch(Exception e){
+                throw new ParseErrorRPCException();
+            }
+        }
+
+        public static String encode(Long obj){
+            if (obj != null){
+                String result = LongConverter.encodeHex(obj);
+                if(checkConstraints(result))
+                    return result;
+                else
+                    throw new ParseErrorRPCException();
+            }
+            else{
+                throw new ParseErrorRPCException();
+            }
+        }
+
+        private static boolean checkConstraints(String s){
+            return regex.matcher(s).find() && s.length() >= 3 && s.length() <= 19;
+        }
+    }
+
+    public static class IntHexStringConverter{
+        private static final Pattern regex = Pattern.compile("^0x[0-9a-fA-F]+");
+
+        public static Integer decode(Object object){
+            try{
+                if (object!=null && checkConstraints(object.toString())){
+                    return IntegerConverter.decode(object);
+                }
+                else{
+                    throw new ParseErrorRPCException();
+                }
+            } catch(Exception e){
+                throw new ParseErrorRPCException();
+            }
+        }
+
+        public static String encode(Integer obj){
+            if (obj != null){
+                String result = IntegerConverter.encodeHex(obj);
+                if(checkConstraints(result))
+                    return result;
+                else
+                    throw new ParseErrorRPCException();
+            }
+            else{
+                throw new ParseErrorRPCException();
+            }
+        }
+
+        private static boolean checkConstraints(String s){
+            return regex.matcher(s).find() && s.length() >= 3 && s.length() <= 11;
+        }
+    }
+
+    public static class AddressConverter{
+        private static final Pattern regex = Pattern.compile(".*");
+
+        public static String decode(Object object){
+            try{
+                if (object!=null && checkConstraints(object.toString())){
+                    return DataHexStringConverter.decode(object);
+                }
+                else{
+                    throw new ParseErrorRPCException();
+                }
+            } catch(Exception e){
+                throw new ParseErrorRPCException();
+            }
+        }
+
+        public static String encode(String obj){
+            if (obj != null){
+                String result = DataHexStringConverter.encode(obj);
+                if(checkConstraints(result))
+                    return result;
+                else
+                    throw new ParseErrorRPCException();
+            }
+            else{
+                throw new ParseErrorRPCException();
+            }
+        }
+
+        private static boolean checkConstraints(String s){
+            return regex.matcher(s).find() && s.length() >= 66 && s.length() <= 66;
```

Comment:
Same here

Reply:
These limits may be derived from constants in the spec file since this is generated source.